src/mono/wasm/build/WasmApp.targets

```diff
@@ -336,6 +336,8 @@
                                Dependencies="$(_WasmPInvokeHPath);$(_WasmPInvokeTablePath)" />
       <_WasmRuntimePackSrcFile Include="$(_WasmRuntimePackSrcDir)driver.c"
                                Dependencies="@(_DriverCDependencies)" />
+      <_WasmRuntimePackSrcFile Include="$(_WasmRuntimePackSrcDir)runtime.c"
+                               Dependencies="@(_DriverCDependencies)" />
```

Comment:
Any other dependencies to add for this?

Reply:
It contains code from driver.c so the same deps should suffice. Probably driver.c needs less deps now.